extension/popup.html

```diff
@@ -31,6 +31,7 @@
       -
       - [1]: https://developer.chrome.com/extensions/contentSecurityPolicy
      -->
+    <script src="jquery-3.1.1.min.js"></script>
```

Comment:
At the moment, I don't see a need for jQuery. It's a rather large library for just making one query.